packages/process/src/node/raw-process.ts

```diff
@@ -69,10 +69,19 @@ export class RawProcess extends Process {
            normalize the error handling by calling the error handler
            instead.  */
         try {
-            this.process = spawn(
-                options.command,
-                options.args,
-                options.options);
+            if (options.command === 'node' && options.args) {
+                this.process = fork(
+                    options.args[0],
+                    [],
+                    {
+                        'stdio': ['pipe', 'pipe', 2, 'ipc']
```

Comment:
This one looks quite hardcoded. It will be difficult to push upstream

Reply:
I don't want to upstream this. The 'proper' fix will require exposing the ability to spawn or fork any process and will touch a lot of the codebase.